security/src/main/java/org/kaazing/gateway/security/auth/BaseStateDrivenLoginModule.java

```diff
@@ -141,5 +153,18 @@ public boolean logout() throws LoginException {
 
     protected abstract boolean doLogout() throws LoginException;
 
+    private LoginResult getLoginResultFromCallback() {
+        LoginResultCallback loginResultCallback = new LoginResultCallback();
+
+        try {
+            this.handler.handle(new Callback[]{loginResultCallback});
+        } catch (IOException|UnsupportedCallbackException e) {
+            LOGGER.debug("Encountered exception handling loginResultCallback", e);
```

Comment:
loginResultCallback should be capitalized "LoginResultCallback", so user knows you are referring to the class. http://grepcode.com/file/repo1.maven.org/maven2/org.kaazing/gateway.server.spi/5.0.1.23/org/kaazing/gateway/server/spi/security/LoginResultCallback.java